core/src/main/scala/kafka/docker/KafkaDockerWrapper.scala

```diff
@@ -52,7 +53,14 @@ object KafkaDockerWrapper extends Logging {
         }
 
         val formatCmd = formatStorageCmd(finalConfigsPath, envVars)
-        StorageTool.main(formatCmd)
+        try {
+          StorageTool.main(formatCmd)
+        } catch {
+          case terseFailure: TerseFailure => if (terseFailure.getMessage.contains(QuorumConfig.QUORUM_VOTERS_CONFIG)) {
+            throw new TerseFailure("The Docker image does not support Dynamic Quorum yet.")
```

Comment:
this comment is too short. Maybe we should say "To maximize compatibility, the Docker image continues to use static voters, which are supported in 3.7 and later."